codepile/ai4code_kaggle/ai4code_kaggle.py

```diff
@@ -0,0 +1,31 @@
+import os
+import gdown
+import pandas as pd
+from codepile.dataset import Dataset
+
+AI4CODE_KAGGLE_URL = "https://drive.google.com/file/d/1RgiY7_zvLPdMPOXaOL4aE-Or5KuZXQ02/view?usp=sharing"
```

Comment:
GDrive links are volatile. Is this your own private drive link?

Reply:
Yes I understand, it's totally temporary, I am waiting for @ncoop57 set up an S3 bucket to store all datasets.